docker/aws/ssm_get_parameter.py

```diff
@@ -0,0 +1,13 @@
+import sys
+from sessions import get_session
+
+account_number = sys.argv[1]
+stage = sys.argv[2]
+param_name = sys.argv[3]
+
+boto_session = get_session(account_number, "TDRJenkinsReadParamsRole" + stage.capitalize())
+client = boto_session.client('ssm', region_name='eu-west-2')
+
+parameter = client.get_parameters(Names=[param_name], WithDecryption=True)
+
+print(parameter['Parameters'][0]['Value'])
```

Comment:
Should this return the parameter instead of printing it? (I might be misunderstanding how this Python script is used.)

Reply:
Its printing the value of the parameter to the shell so it can be used by the shell

Reply:
As discussed in person, let's add a comment to this script explaining how to safely execute it without printing sensitive values to the console.

Tom explained to me that we the Jenkins parameter store plugin doesn't look like a good alternative in this case, because Jenkins runs in the management account, but the end-to-end tests that call this Python script will need environment-specific parameters.

Reply:
Added comment to the ssm param script